qemu/tests/virtio_console.py

```diff
@@ -411,7 +411,7 @@ def test_rw_nonblocking_mode():
         try:
             guest_worker.cmd("virt.recv('%s', 10, 1024, False)"
                              % port.name, 10)
-        except qemu_virtio_port.VirtioPortException as details:
+        except qemu_virtio_port.VirtioPortException as details:  # pylint: disable=E1135
```

Comment:
Is it reported by the new inspektor (0.5.2)? I didn't see the error with either the pylint (2.6.0) on my laptop or the travis ci check (https://travis-ci.org/github/autotest/tp-qemu/jobs/755494599).

Reply:
This may the default setting of python3, I can reproduce it on python3 + inspektor-0.5.2 + pylint-2.6.0
```shell
# pylint -E qemu/tests/virtio_console.py 
************* Module qemu.tests.virtio_console
qemu/tests/virtio_console.py:415:64: E1135: Value 'details' doesn't support membership test (unsupported-membership-test)
qemu/tests/virtio_console.py:1103:8: E0401: Unable to import 'autotest.client' (import-error)
```

```shell
# pip list | grep -E 'pylint|inspektor'
inspektor                            0.4.5
pylint                               2.6.0
# python3 -V
Python 3.6.8
# inspekt checkall --disable-style E501,E265,W601,E402,E722,E741 --no-license-check
......
......
************* Module qemu.tests.virtio_console
E1135:415,64: run.test_rw_nonblocking_mode: Value 'details' doesn't support membership test
Lint: /var/lib/avocado/data/avocado-vt/virttest/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/virtio_console.py FAIL 
Global check FAIL
```

And refer to https://pycodequ.al/docs/pylint-messages/e1135-unsupported-membership-test.html, `VirtioPortException` class hasn't membership protocol `(__contains__/__iter__/__getitem__)`.

Reply:
Got it, then suggest the fix:
```patch
- if '[Errno 11] Resource temporarily unavailable' in details:
+ if '[Errno 11] Resource temporarily unavailable' in str(details):
```

Reply:
ok.